heclib/heclib_c/src/Internal/zCopyRecord6Dynamic.c

```diff
@@ -0,0 +1,81 @@
+#include <heclib.h>
+#include <string.h>
+
+#define KIHEAD 1000
+#define KCHEAD 100
+
+#ifdef _MSC_VER
+char* strndup(const char* src, size_t max_len) {

```

Comment:
could we use the method stringFortToC instead?     

note: There is a bug in stringFortToC  where it doesn't trim more than one trailing space.

Reply:
We could, but shouldn't we fix the bug first? I usually trust the C library functions when they exist - it's just that for some reason Microsoft doesn't include this function anymore.

Reply:
Didn't find any bug in `stringFortToC()`. It only nulls one character, but it only allocates one more than the number of non-trailing-space characters of the Fortran string, so one is enough.